calc_app/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+
+    def __str__(self):
+        return self.name
+
+
+class ExpenseItem(models.Model):
+    category = models.ForeignKey(Category, on_delete=models.CASCADE, null=True)
+    cost = models.PositiveIntegerField()
+    date = models.DateField()
```

Comment:
It would be great to enter date from some "calendar" or have a predefined way to write a date

Reply:
Also, way of creating expense enrty is inobvious. We should create a better UX for the customers

Reply:
> 

I've understood about the calendar, but I'm not sure I understood you about the non-obvious expense entry creation. May be it is better to make another interface(webpage) that will be responsible for such operations?

Reply:
It makes sense. Or separate button like "Create expense" and just represent categories as non-clickable stuff